bleak/backends/p4android/recipes/bleak/__init__.py

```diff
@@ -1,27 +1,40 @@
+import os
+
 from pythonforandroid.recipe import PythonRecipe
 from pythonforandroid.toolchain import shprint, info
 import sh
 from os.path import join
 
 
 class BleakRecipe(PythonRecipe):
-    version = None
-    url = None
+    version = '0.20.2'
```

Comment:
Can we do something else so that we don't have to update this version on every release?